src/integration/java/com/smartcar/sdk/IntegrationTest.java

```diff
@@ -38,6 +38,8 @@
             "read_location",
             "read_odometer",
             "read_fuel",
+            "read_tires",
+            "read_engine_oil",
```

Comment:
missing tests for this 2 functions scopes, you can add the tests on the VehicleTest.java folder

Reply:
i added these